packages/block-library/src/footnotes/index.js

```diff
@@ -21,9 +21,8 @@ export const settings = {
 	edit,
 };
 
-// Would be good to remove the format and HoR if the block is unregistered.
-registerFormatType( formatName, format );
-
 export const init = () => {
+	// Would be good to remove the format and HoR if the block is unregistered.
```

Comment:
The bit about the HoR can be remove, forgot that when we change how it's implemented.